openedx/features/content_tagging/rest_api/v1/tests/test_views.py

```diff
@@ -0,0 +1,637 @@
+        )
+        self.st2 = Taxonomy.objects.create(name="st2", enabled=False)
+        self.st2.taxonomy_class = SystemDefinedTaxonomy
+        self.st2.save()
+        TaxonomyOrg.objects.create(
+            taxonomy=self.st2,
+            rel_type=TaxonomyOrg.RelType.OWNER,
+        )
+
+        # Global taxonomy
+        self.t1 = Taxonomy.objects.create(name="t1", enabled=True)
+        TaxonomyOrg.objects.create(
+            taxonomy=self.t1,
+            rel_type=TaxonomyOrg.RelType.OWNER,
+        )
+        self.t2 = Taxonomy.objects.create(name="t2", enabled=False)
+        TaxonomyOrg.objects.create(
+            taxonomy=self.t2,
+            rel_type=TaxonomyOrg.RelType.OWNER,
+        )
+
+        # OrgA taxonomy
+        self.tA1 = Taxonomy.objects.create(name="tA1", enabled=True)
+        TaxonomyOrg.objects.create(
+            taxonomy=self.tA1,
+            org=self.orgA,
+            rel_type=TaxonomyOrg.RelType.OWNER,
+        )
+        self.tA2 = Taxonomy.objects.create(name="tA2", enabled=False)
+        TaxonomyOrg.objects.create(
+            taxonomy=self.tA2,
+            org=self.orgA,
+            rel_type=TaxonomyOrg.RelType.OWNER,
+        )
+
+        # OrgB taxonomy
+        self.tB1 = Taxonomy.objects.create(name="tB1", enabled=True)
+        TaxonomyOrg.objects.create(
+            taxonomy=self.tB1,
+            org=self.orgB,
+            rel_type=TaxonomyOrg.RelType.OWNER,
+        )
+        self.tB2 = Taxonomy.objects.create(name="tB2", enabled=False)
+        TaxonomyOrg.objects.create(
+            taxonomy=self.tB2,
+            org=self.orgB,
+            rel_type=TaxonomyOrg.RelType.OWNER,
+        )
+
+        # OrgA and OrgB taxonomy
+        self.tC1 = Taxonomy.objects.create(name="tC1", enabled=True)
+        TaxonomyOrg.objects.create(
+            taxonomy=self.tC1,
+            org=self.orgA,
+            rel_type=TaxonomyOrg.RelType.OWNER,
+        )
+        TaxonomyOrg.objects.create(
+            taxonomy=self.tC1,
+            org=self.orgB,
+            rel_type=TaxonomyOrg.RelType.OWNER,
+        )
+        self.tC2 = Taxonomy.objects.create(name="tC2", enabled=False)
+        TaxonomyOrg.objects.create(
+            taxonomy=self.tC2,
+            org=self.orgA,
+            rel_type=TaxonomyOrg.RelType.OWNER,
+        )
+        TaxonomyOrg.objects.create(
+            taxonomy=self.tC2,
+            org=self.orgB,
+            rel_type=TaxonomyOrg.RelType.OWNER,
+        )
+
+
+@ddt.ddt
+@override_settings(FEATURES={"ENABLE_CREATOR_GROUP": True})
+class TestTaxonomyViewSet(TestTaxonomyViewSetMixin, APITestCase):
+    """
+    Test cases for TaxonomyViewSet when ENABLE_CREATOR_GROUP is True
+    """
+
+    @ddt.data(
+        ("user", None, None, ("ot1", "st1", "t1", "tA1", "tB1", "tC1")),
+        ("userA", None, None, ("ot1", "st1", "t1", "tA1", "tB1", "tC1")),
+        ("userS", None, None, ("ot1", "ot2", "st1", "st2", "t1", "t2", "tA1", "tA2", "tB1", "tB2", "tC1", "tC2")),
```

Comment:
Since the new default pagination has a `page_size = 10`, we don't get the final `tC1` and `tC2` taxonomies in this list, so this test fails (in both TestCase classes).

It's ok to just remove these  -- when we specify `orgA` below, we see that these taxonomies are returned.

```suggestion
        # Default page_size=10, and so "tC1", "tC2" appear on page 2.
        ("userS", None, None, ("ot1", "ot2", "st1", "st2", "t1", "t2", "tA1", "tA2", "tB1", "tB2")),
```

Reply:
Thank you @pomegranited ! I missed this failed test.
This is fixed now!